### ISSUE TITLE
Chrome browser issue unsafe http:// URLs

### ISSUE THREAD

**Issue by srcatto** [csev/py4e#401]
Chrome Version 111.0.5563.65 (Official Build) (64-bit)
https://github.com/csev/py4e/blob/master/tools/python-data/regex_sum.php
Example - 
Sample data: http://py4e-data.dr-chuck.net/regex_sum_42.txt (There are 90 values with a sum=445833)
Actual data: http://py4e-data.dr-chuck.net/regex_sum_1089727.txt (There are 92 values and the sum ends with 895)

The 2 links, lines 90 & 92 have a deHttps(). In this case Chrome does, 
(filename) can't be downloaded securely [Discard] button. Click on the '^' and select 'Keep' to download file. Otherwise clicking link opens as text (in browser) and relies on a good copy / paste into a new file or Save Link As, Save.

Instead of editing instructions, it is easier to use https:// URLs

Basically same in Firefox 111.0 x64, 'File not downloaded potential security risk', click '>'  then 'Allow download'.

Edge, same Version 111.0.1661.41. Save link as, Save, click '...' , Keep, Keep anyway.

**Comment by csev** [comment]
Thank you - this is fixed in the latest version.
